src/async-wrap.cc

```diff
@@ -428,7 +428,8 @@ void AsyncWrap::ClearIdStack(const FunctionCallbackInfo<Value>& args) {
 void AsyncWrap::AsyncReset(const FunctionCallbackInfo<Value>& args) {
   AsyncWrap* wrap;
   ASSIGN_OR_RETURN_UNWRAP(&wrap, args.Holder());
-  wrap->AsyncReset();
+  double eid = args[0]->IsNumber() ? args[0]->NumberValue() : -1;
```

Comment:
Could you use a more verbose name instead of `eid`?

Reply:
but `execution_async_id` is a bit long. would `exec_id` be good enough?

Reply:
`exec_id` is great.

Reply:
Imho, `execution_async_id` is great, too – let’s choose verbose names whenever we can, a *lot* more people need to understand rather the code than need to write it.